src/imgix-core-js.js

```diff
@@ -62,8 +61,17 @@
           this.settings.domains[0] = this.settings.host;
       }
 
+      this.settings.domains.forEach(function(domain) {
+        if(constants.DOMAIN_REGEX.exec(domain) == null){
```

Comment:
[nit] Codestyle here dictates there should be spaces around the parens: `if (...) {`